packages/cron/README.md

```diff
@@ -0,0 +1,105 @@
+# Create a Cron Job with Functionless
+
+In this example, we'll create a Cron job.
+
+The Cron Job will use [AWS EventBridge's](https://aws.amazon.com/eventbridge/) scheduled rules to trigger a job in a lambda [Lambda Functions](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html).
+
+Then we'll deploy and test the application.
+
+To see the completely application, go [here](https://github.com/functionless/functionless-samples/packages/cron).
+
+## Create an new app
+
+> yarn create functionless
```

Comment:
The example code doesn't use this - should we make sure it does?

Also this isn't a code block

Reply:
Was going to ask if the code should be using the template. It would be nice to keep dogfooding the template just to discover if anything is off.